concordia/templates/base.html

```diff
@@ -171,17 +171,13 @@ <h2>Follow Us</h2>
     {% endblock site-footer %}
 
     {% if SENTRY_FRONTEND_DSN %}
-        <script src="https://cdnjs.cloudflare.com/ajax/libs/raven.js/3.26.1/raven.min.js" integrity="sha256-N3YzTdAC27tvNKpIM2rly/WvlJf8YAat0NjPXgh5Bw4=" crossorigin="anonymous"></script>
+        <script src="https://browser.sentry-cdn.com/5.0.8/bundle.min.js" integrity="sha384-PKOJCSVL6suo2Qz9Hs4hkrZqX7S6iLwadxXxBEa0h0ycsuoDNZCiAcHlPGHYxU6l" crossorigin="anonymous"></script>
         <script>
-            if (navigator.userAgent.indexOf("CloudFlare-AlwaysOnline") < 0) {
-                Raven.config("{{ SENTRY_FRONTEND_DSN }}", {
-                    environment: "{{ CONCORDIA_ENVIRONMENT }}",
-                    release: "{{ RAVEN_CONFIG.release }}",
-                    ignoreUrls: [
-                        /^moz-extension/
-                    ]
-                }).install();
-            }
+            Sentry.init({
```

Comment:
I _think_ we're good with the filters for known crawlers but we should confirm that this doesn't spam us with the security restrictions for the CloudFlare Always Online crawler.

Reply:
I added the condition back in - I didn't see anything in the docs about filters for known crawlers. Do you have any reference about it I could look into?